discord/cog.py

```diff
@@ -23,7 +23,8 @@
 DEALINGS IN THE SOFTWARE.
 """
 from __future__ import annotations
-
```

Comment:
This space (between `from __future__ import annotations` and `import importlib`/`import os`) should still be here.